addon/maptools/maptool.js

```diff
@@ -5,6 +5,10 @@ export default Ember.Object.extend({
 
   map: null,
 
+  isMultiTool() {
+    return false;
+  },
```

Comment:
In my opinion on class inheritance it's better methods overriding, as properties overriding and reinitializing.